install/runners/nunit-tools.wxs

```diff
@@ -105,11 +104,15 @@
                       ProcessorArchitecture="msil" 
                       Source="$(var.BuildDir)nunit.engine.dll" />
 			</Component>
-            <Component Id="NUNIT.ENGINE.ADDINS">
-                <File Id="nunit.engine.addins"
+			<Component Id="NUNIT.ENGINE.ADDINS">
+				<File Id="nunit.engine.addins"
                       Source="$(var.BuildDir)nunit.engine.addins" />
-            </Component>
-            <Component Id="NUNIT.ENGINE.API">
+			</Component>
```

Comment:
Right. I'm afraid I'm not at all used to using VS to edit XML. In the past I have typically done all the wxs and nant XML scripts in a text editor, inserting the number of spaces I want manually.

I'm OK with the two-space standard, although I sometimes break up long lines and line up the attributes for clarity. We can deal with that when we come to it.